gammapy/modeling/models/spatial.py

```diff
@@ -602,8 +606,10 @@ def read(cls, filename, normalize=True, **kwargs):
             Keyword arguments passed to `Map.read()`.
         """
         m = Map.read(filename, **kwargs)
+
```

Comment:
@adonath
sorry I add the black lines to have the diff here. I don't know about that in the read and from_dict() method. I don't like too much to force a default unit if there is none? Should we remove those in read and from_dict() ?

Reply:
How about checking that the unit is equivalent to `sr-1` and if not add a `log.warning(Spatial template unit is not equivalent to sr^-1. Changing unit to sr^-1)`?